functions/postSurveyInit.ts

```diff
@@ -89,6 +94,27 @@ const triggerPostSurveyFlow = async (context: Context<EnvVars>, channelSid: stri
   return messageResult;
 };
 
+const getTriggerMessage = (event: Body): string => {
+  // Try to retrieve the triggerMessage for the approapriate language (if any)
+  const { taskLanguage } = event;
+  if (taskLanguage) {
+    try {
+      const translation = JSON.parse(
+        Runtime.getAssets()[`/translations/${taskLanguage}/postSurveyMessages.json`].open(),
+      );
+
+      if (!translation.triggerMessage)
+        console.error(`triggerMessage key is missing in translation for ${taskLanguage}`);
+
+      return translation.triggerMessage;
+    } catch {
+      console.error(`Couldn't retrieve triggerMessage translation for ${taskLanguage}`);
+    }
+  }
+
+  return 'Hey! Before you leave, can you answer a few questions about this contact?';
```

Comment:
This message could come from the assets en-US file, instead of hardcoded.

Reply:
I think at some point is better to fallback to something in the code, just in case we can't access the assets and still be able to trigger the post survey. I will merge as it is for now but if you guys think this is better I will open a subsequent PR to address it.